src/components/SignName.vue

```diff
@@ -10,6 +10,7 @@
           label="Full Name"
           placeholder="John Doe"
           required
+          @keyup="checkFormFilled"
```

Comment:
It is an interesting and clever solution to check on every keystroke if the form is filled. I don't necessarily think this needs to be changed, but consider that every method call here incurs a performance penalty (even if it is a small one).